system-test/storage.ts

```diff
@@ -2143,22 +2143,19 @@ describe('storage', () => {
       };
 
       const file = bucket.file('large-metadata-error-test');
-      // Save file with metadata size more then 2MB.
```

Comment:
Please switch this back to an `assert.rejects`.  When you try to do an assert in the catch, there's a chance the method just doesn't throw.  The `await assert.rejects` is careful about making sure both that it throws, and that it gets the right error message.

Reply:
I was wondering about that. I saw the try/catch used in other tests, so we may have some more work to do across the libraries to be safe.

Reply:
Yeah, I try to call em out when I see them.  Things were messy until `assert.rejects` made it's way consistently into node core.

Reply:
@JustinBeckwith  Thanks for bringing this up.